src/System.Net.Primitives/tests/FunctionalTests/CredentialCacheTest.cs

```diff
@@ -22,7 +22,14 @@ public static class CredentialCacheTest
 
         private static readonly string authenticationType1 = "authenticationType1";
         private static readonly string authenticationType2 = "authenticationType2";
+        
+        private static readonly string authenticationTypeNTLM = "NTLM";
+        private static readonly string authenticationTypeKerberos = "Kerberos";
+        private static readonly string authenticationTypeNegotiate = "Negotiate";
+        private static readonly string authenticationTypeBasic = "Basic";
+        private static readonly string authenticationTypeDigest = "Digest";
 
+        private static readonly NetworkCredential customCredential = new NetworkCredential("username", "password");
         private static readonly NetworkCredential credential1 = new NetworkCredential("username1", "password");
         private static readonly NetworkCredential credential2 = new NetworkCredential("username2", "password");
         private static readonly NetworkCredential credential3 = new NetworkCredential("username3", "password");
```

Comment:
Now that you have created the 'customerCredential' that has username/password.  why have the credential1 thru credential3 variables defined?